src/Reflection/Adapter/ReflectionNamedType.php

```diff
@@ -23,7 +25,7 @@ public function getName(): string
 
     public function __toString(): string
     {
-        return ($this->allowsNull ? '?' : '')
+        return ($this->allowsNull && strtolower($this->betterReflectionType->getName()) !== 'mixed' ? '?' : '')
```

Comment:
This is kinda missing a test, but meh :P

Reply:
Some tests failed with not expecting `?mixed` without this change :)

Reply:
All good then :D :muscle: